### PR TITLE
refactor extended resource with `kubernetes.io` namespace

### PR DESCRIPTION
**Pull request by zwzhang0107**
Signed-off-by: 佑祎 <zzw261520@alibaba-inc.com>

### Ⅱ. Does this pull request fix one issue?
#83

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: docs/proposals/api-machinery/20220707-rename-extended-resource.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+2. Using `koordinato.sh/` namespace for koordinator batch resources.
+3. Using `koordinato.sh/` for resource requests and annotations for resource limits.
+
+| Alternative | Pros | Cons |
+| --------------- | ---- | ---- |
+| annotations["batch-cpu"] | resource overcommitment; better expression (cpu=1.5cores); | incompatible with k8s scheduler |
+| extended resource `koordinator.sh/batch-cpu` |  compatible with k8s scheduler | resource overcommitment is not allowed; bad expression (cpu=1500 milli-cores) | 
+| setting requests as extended resource `koordinator.sh/batch-cpu` and limits in pod annotations | barely works with ugly design | bad expression (cpu=1500 milli-cores); confusing using habits |
+
+## Unsolved Problems
```
Comment (hormes) [inline docs/proposals/api-machinery/20220707-rename-extended-resource.md:75]: How to upgrade the old koordinator version to the new one?

Comment (zwzhang0107) [reply docs/proposals/api-machinery/20220707-rename-extended-resource.md:77]: koord-scheduler will provide a filter plugin to support pods with old and new format in next few versions.

File: docs/proposals/api-machinery/20220707-rename-extended-resource.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  limites:
+    koordinator.kubernetes.io/batch-cpu: 3 # 3 core
+    koordinator.kubernetes.io/batch-memory: 2Gi    
+```
+
+### Implementation
+1. Defines new formats of extended resources as `koordinator.kubernetes.io/batch-cpu` and `koordinator.kubernetes.io/batch-memory`
+2. Koord-manager updates new extended resources to `Node.Status`. 
+3. Webhook of `ColocationProfile` injects new extended resources to `Pod`. 
+4. Considering pods with the old format may continuously exist for a period of time, koord-scheduler will provide a
```
[the remark this answers is not in the document]
Comment (eahydra) [reply docs/proposals/api-machinery/20220707-rename-extended-resource.md:62]: I forgot the built-in plugin `resourceFit`.. Really need to develop a new plugin....

**Pull request merged**